cmds/mount/mount.go

```diff
@@ -25,14 +27,48 @@ var (
 	opt    = flag.String("o", "", "Specify mount options")
 )
 
+// extended from boot.go
+func getSupportedFilesystem(originFS string) ([]string, bool, error) {
+	var known bool
+	var err error
+	fs, err := ioutil.ReadFile("/proc/filesystems")
```

Comment:
I don't think this does what you think?

On my system, /proc/filesystems contains:

```
nodev   sysfs
nodev   rootfs
nodev   ramfs
nodev   bdev
nodev   proc
nodev   cpuset
nodev   cgroup
nodev   cgroup2
nodev   tmpfs
nodev   devtmpfs
nodev   debugfs
nodev   tracefs
nodev   securityfs
nodev   sockfs
nodev   bpf
nodev   pipefs
nodev   hugetlbfs
nodev   devpts
nodev   pstore
nodev   mqueue
        ext3
        ext2
        ext4
nodev   autofs
nodev   rpc_pipefs
nodev   binfmt_misc
        fuseblk
nodev   fuse
nodev   fusectl
```

`n[0]` is going to be `nodev` for most of them.

Reply:
Good point! I just used the boot.go function naively. mount command need to be able to mount FS that are nodev.... Correcting...

Reply:
I hope I properly committed the changes this time.... let's see if that is reviewable...